app/nagbot.py

```diff
@@ -108,37 +106,36 @@ def execute_internal(channel, dryrun):
                                       and r.is_safe_to_stop(today_date=TODAY_YYYY_MM_DD)))
 
             if len(resources_to_terminate) > 0:
-                message = 'I terminated the following {}s: '.format(ec2_type)
+                message = f'I terminated the following {ec2_type}s: '
                 for r in resources_to_terminate:
                     contact = sqslack.lookup_user_by_email(r.contact)
                     message = message + r.make_resource_summary() + \
-                        ', "Terminate after"={}, "Monthly Price"={}, Contact={}\n' \
-                        .format(r.terminate_after, money_to_string(r.monthly_price), contact)
+                        f', "Terminate after"={r.terminate_after}, "Monthly Price"=' \
+                        f'{money_to_string(r.monthly_price)}, Contact={contact}\n'
                     r.terminate_resource(dryrun=dryrun)
                 sqslack.send_message(channel, message)
             else:
-                sqslack.send_message(channel, 'No {}s were terminated today.'
-                                     .format(ec2_type))
-
-            if len(resources_to_stop) > 0:
-                message = 'I stopped the following {}s: '.format(ec2_type)
-                for r in resources_to_stop:
-                    contact = sqslack.lookup_user_by_email(r.contact)
-                    message = message + r.make_resource_summary() + \
-                        ', "Stop after"={}, "Monthly Price"={}, Contact={}\n' \
-                        .format(r.stop_after, money_to_string(r.monthly_price), contact)
-                    resource.stop_resource(r.region_name, r.resource_id, dryrun=dryrun)
-                    resource.set_tag(r.region_name, r.ec2_type, r.resource_id, r.nagbot_state_tag_name, 'Stopped on '
-                                     + TODAY_YYYY_MM_DD, dryrun=dryrun)
-                sqslack.send_message(channel, message)
-            else:
-                sqslack.send_message(channel, 'No {}s were stopped today.'.format(ec2_type))
+                sqslack.send_message(channel, f'No {ec2_type}s were terminated today.')
+
+            if resource_type == "Instance":
```

Comment:
I implemented this because the output on slack includes "No volumes are due to be stopped at this time" and since Volumes, Snapshots, and AMIs don't stop I thought it would look cleaner on slack to only display _stopped_ AMIs.

Reply:
That seems like a good idea. Maybe there is a way to define if a resource type is "stopable" generically?

Reply:
I see, so we would be

`if resource_type.is_stoppable()`

rather than 

`if resource_type == "Instance"`

I can implement that

Reply:
done